app/src/main/java/com/hover/stax/fieldstates/FieldState.java

```diff
@@ -0,0 +1,27 @@
+package com.hover.stax.fieldstates;
+
+public class FieldState {
```

Comment:
It should. Will fix in next commit

Reply:
I went through this again. I thought of this object to serve both the CustomDropdown and CustomTextInputLayout. Reason why both dropdown and textinput isn't using the same custom layout is because I haven't found a way to set styling programmatically for it.